src/uiTest/scala/com/virtuslab/gitmachete/uitest/UITestSuite.scala

```diff
@@ -97,9 +98,10 @@ class UITestSuite extends BaseGitRepositoryBackedIntegrationTestSuite(SETUP_WITH
   @Test def discoverBranchLayout(): Unit = {
     // When model is refreshed and machete file is has not been modified for a long time, then discover suggestion should occur
     setLastModifiedDateOfMacheteFileToEpochStart()
-    project.openGitMacheteTab()
-    project.acceptSuggestedBranchLayout()
-    probe.await()
+    doAndAwait {
```

Comment:
In contexts like this (where `probe.await()` happens between one sequence of operations and another, I'd leave as it was